### ISSUE TITLE
Comments

### ISSUE THREAD

**Issue by chemistry-sourabh** [CCI-MOC/ims#45]
A Lot of comments and doc strings must be added to help new comers understand code easily.

**Comment by ravisantoshgudimetla** [comment]
+100 and I think, its part of PEP 257.

**Comment by apoorvemohan** [comment]
@chemistry-sourabh are you working on this?

**Comment by naved001** [comment]
yeah, with pylint enabled you'll be forced to add a docstring at least.

**Comment by apoorvemohan** [comment]
@chemistry-sourabh can we close this?
